### PR TITLE
Refactor AutoCloudConsumer

### PR DESCRIPTION
**Pull request by yahzaa**

### CODE REVIEW

File: autocloud/consumer.py [hunk 1]
```diff
@@ -51,58 +51,49 @@ def consume(self, msg):
...
-                            absolute_path = '{}/{}'.format(location,
-                                                           relative_path)
-
-                            item.update({
-                                'compose': compose_details,
-                                'absolute_path': absolute_path,
-                            })
-                            images.append(item)
-                            compose_db_update = True
+        else:
```
Comment (sayanchowdhury) [inline autocloud/consumer.py]: We can just remove the `else` block here and set compose_images_json to None in the beginning

File: autocloud/consumer.py [hunk 2]
```diff
@@ -51,58 +51,49 @@ def consume(self, msg):
...
+                            'absolute_path': absolute_path,
+                        })
+                        images.append(item)
+                        compose_db_update = True
 
             if compose_db_update:
                 session = init_model()
-                compose_date = datetime.strptime(compose_details['date'],
-                                                 '%Y%m%d')
+                compose_date = datetime.strptime(compose_details['date'], '%Y%m%d')
```
Comment (sayanchowdhury) [inline autocloud/consumer.py:95]: Will this not break PEP8?

File: autocloud/consumer.py [hunk 3]
```diff
@@ -51,58 +51,49 @@ def consume(self, msg):
...
-                            item.update({
-                                'compose': compose_details,
-                                'absolute_path': absolute_path,
-                            })
-                            images.append(item)
-                            compose_db_update = True
+        else:
+            compose_images_json = None
+
+        if compose_images_json:
```
Comment (sayanchowdhury) [inline autocloud/consumer.py]: `NoneType` test should be done as `compose_images_json is not None`

**Review by sayanchowdhury — CHANGES_REQUESTED** [review]

File: autocloud/consumer.py (hunk 2 above)
Comment (yahzaa) [reply autocloud/consumer.py:95]: no, it's less than 80 cols.

**Comment by yahzaa** [comment]
updated commit

**Review by sayanchowdhury — APPROVED** [review]

**Pull request merged**